dotnetAvalonia/Views/MainWindow.axaml

```diff
@@ -11,6 +11,8 @@
         Title="__change__">
 
     <Design.DataContext>
+        <!-- This only sets the DataContext for the previewer in an IDE,
+             to set the actual DataContext for runtime, set the DataContext property in code (look at App.axaml.cs) -->
```

Comment:
So, the person should set the `DataContext` property at `App.axaml.cs` ? I am a bit confused with the sentence, but this may just be me as I am not familiarized with `DataContext`